solr/core/src/java/org/apache/solr/handler/component/HighlightComponent.java

```diff
@@ -212,27 +212,31 @@ public void handleResponses(ResponseBuilder rb, ShardRequest sreq) {}
 
   @Override
   public void finishStage(ResponseBuilder rb) {
```

Comment:
'Hide whitespace' diff mode makes it easier to see that the changes to this method are small (and _extremely_ subtle, should add comments really, later).